index.js

```diff
@@ -21,7 +23,55 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+   /* THIS IS ITERATION 2!!!!!!!!!
+    Recipe.create( data[0] )
+      .then((recipe) => {
+        console.log(recipe.title)
+
+      })
+      .catch(()=> {
+        console.log()
```

Comment:
don't console.log the nothingness. Instead catch and error and console.log that :p